plugins/modules/iosxr_vrf_global.py

```diff
@@ -0,0 +1,606 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2024 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The module file for iosxr_vrf_global
+"""
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: iosxr_vrf_global
+short_description: Manages global VRF configuration.
+description:
+  - This module manages VRF configurations on Cisco IOS-XR devices.
+  - It enables playbooks to handle either individual VRFs or the complete VRF collection.
+  - It also permits removing non-explicitly stated VRF definitions from the setup.
+version_added: 9.0.0
+author: Ruchi Pakhle (@Ruchip16)
+notes:
+  - Tested against Cisco IOS-XR Version 9.0.0
+  - This module works with connection C(network_cli). See L(the IOS_XR Platform Options,../network/user_guide/platform_iosxr.html)
```

Comment:
Relative links don't work anymore after ansible to ansible-core split. Please add full URL to the platform_guide.rst residing in this collection now.